R/ids_with_token.R

```diff
@@ -15,11 +15,16 @@
 #'
 #' @param fun For additional flexibility, a function to search for in
 #' the `token` column of `parsed_content`. Typically `==` or `%in%`.
+#' @param source_file [DEPRECATED] Same as `source_expression`. Will be removed.
 #' @return `ids_with_token`: The indices of the `parsed_content` data frame
 #' entry of the list of source expressions. Indices correspond to the
 #' *rows* where `fun` evaluates to `TRUE` for the `value` in the *token* column.
 #' @export
-ids_with_token <- function(source_expression, value, fun = `==`) {
+ids_with_token <- function(source_expression, value, fun = `==`, source_file) {
```

Comment:
Can we add tests for the warning, like we did for `semicolon_terminator_linter()`?

Reply:
would prefer as a follow-up -- will get buried in this huge PR.